content/4_2_rethink_db.tex

```diff
@@ -0,0 +1,412 @@
+\chapter{Document Oriented}
+\section{RethinkDB}
+\chapterauthor{Anne Born, Dorian Czichotzki}
+
+\subsection{Introduction}
+In today's data-driven world, the availability of information in real-time becomes more and more important. Many applications rely on an analysis and delivery of data in a time-frame that is considered as immediate by the user. Be it multi-player games, real-time analytics or connected devices in an IoT infrastructure \autocite{Wingerath2017}. Data should not only be delivered quickly but also proactively, focusing more on a push-based architecture, giving the responsibility to keep the client informed to the server, instead of the conventional pull-based database were the client has to request information each time it is of interest. \\ One kind of technology, implementing this immediate handling of information are so called real-time databases, which are discussed in this chapter by reference to RethinkDB, an example for such a push-based database.\\
```

Comment:
In general, your LaTeX looks good.

However, please remove all non-necessary forced line breaks (`\\`). This is not how it's meant to be used. This causes ~15 BadBox warnings.